test/lib/k8s/job.js

```diff
@@ -0,0 +1,307 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
```

Comment:
this should be in a helper file that is loaded as a test harness via mocha's [`--require` cli flag](https://mochajs.org/#-require-module-r-module)